src/servers/sapphire_zone/Network/Handlers/PacketHandlers.cpp

```diff
@@ -515,23 +515,23 @@ void Core::Network::GameConnection::chatHandler( const Packets::GamePacket& inPa
    {
    case ChatType::Say:
    {
-      if (player.getGmRank() > 0)
+      if (player.getOnlineStatus() == OnlineStatus::GameMaster || player.getOnlineStatus() == OnlineStatus::GameMaster1 || player.getOnlineStatus() == OnlineStatus::GameMaster2 )
```

Comment:
honestly this probably would be better to have another method inside player which returns true if these are set, cause no doubt there's going to be other stuff down the line that we're gonna want to switch based on whether a player is acting as a gm or not, and the duplication here isnt that great anyway 😛